README.md

```diff
@@ -198,6 +198,98 @@ To add copyright headers to all files with IntelliJ
 2. Create a new copyright profile matching existing copyright header found on all files, name it Dockstore (Settings -> Copyright -> Copyright Profiles -> Add New)
 3. Set the default project copyright to Dockstore (Settings -> Copyright)
 
+### Setting up a Mac with Apple silicon for Dockstore development
+Install IntelliJ (Be sure to click on the dropdown and select '.dmg (Apple Silicon)')
```

Comment:
Suggest the "Be sure..." part be a note and/or rephrased, e.g., if on Apple Silicon, select the .dmg (Apple Silicon), otherwise suggest .dmg (whatever they call the other one).

Also, IntelliJ is not a requirement (I think Steve works without out), so I would move it down below to an optional/recommended section

Reply:
done